docs/SviatoslavTaranenko/Games.html

```diff
@@ -0,0 +1,161 @@
+                        <span class="Games__cards_btm_bold" data-type="Games__cards_btm_bold" data-card-developer>Developer: </span>
+                    </p>
+                    <p class="Games__cards_btm_name" data-type="data-card-release-date">
+                        <span class="Games__cards_btm_bold" data-type="Games__cards_btm_bold" data-card-release-date>Release_date: </span>
+                    </p>
+
+                </div>
+            </div>
+        </template>
+        <footer class="footer">
+            <div class="footer__icons">
+                <div class="footer__icons_row">
+                    <a href="https://www.facebook.com/sviatoslav.taranenko" target="_blank">
+                        <div class="footer__icons_all">
+                            <img src="img/facebook.png" alt="Logo">
+                        </div>
+                    </a>
+                    <a href="https://www.linkedin.com/in/%D1%81%D0%B2%D1%8F%D1%82%D0%BE%D1%81%D0%BB%D0%B0%D0%B2-%D1%82%D0%B0%D1%80%D0%B0%D0%BD%D0%B5%D0%BD%D0%BA%D0%BE-6bb602273/" target="_blank">
+                        <div class="footer__icons_all">
+                            <img src="img/LinkedIn.png" alt="Logo">
+                        </div>
+                    </a>
+                    <a href="https://www.instagram.com/sviat_taranenko" target="_blank">
+                        <div class="footer__icons_all">
+                            <img src="img/Instagram.png" alt="Logo">
+                        </div>
+                    </a>
+                </div>
+            </div>
+            <div class="footer__text">
+                <p>Made with 💗 on course
+                    <a class="about__text_link" href="https://www.mastersacademy.education/frontend-for-beginners-it" target="_blank">'Frontend for beginners' from Masters Academy in 2023,</a>  by Sviatoslav Taranenko
+                </p>
+            </div>
+        </footer>
+        <script src="scripts/Games.js"></script>
+        <script>
+            document.addEventListener('DOMContentLoaded', () => {
+                loadAndRenderGames();
+            });
+
+            async function loadAndRenderGames() {
+                const loadingOverlay = document.getElementById('loading-overlay');
+                loadingOverlay.style.display = 'flex';
+
+                try {
+                    const response = await fetch('https://mmo-games.p.rapidapi.com/games', {
+                        method: 'GET',
+                        headers: {
+                            'X-RapidAPI-Key': '1c3169c707mshb51bff34cbc9ff6p1749b9jsn648a19134256',
+                            'X-RapidAPI-Host': 'mmo-games.p.rapidapi.com',
+                        },
+                    });
+
+                    const gamesData = await response.json();
+
+                    renderGames(gamesData.slice(0, 50));
+                } catch (error) {
+                    console.error('Помилка при отриманні ігор:', error);
```

Comment:
```suggestion
                    console.error('Games loading error:', error);
```
All your project is written in English 💁‍♂️